packages/phrase/src/pull-translations.test.ts

```diff
@@ -33,21 +35,21 @@ describe('pull translations', () => {
     beforeEach(() => {
       (pullAllTranslations as jest.Mock).mockClear();
       (writeFile as jest.Mock).mockClear();
-    });
-    (pullAllTranslations as jest.Mock).mockImplementation(() =>
-      Promise.resolve({
-        en: {
-          'hello.mytranslations': {
-            message: 'Hi there',
+      (pullAllTranslations as jest.Mock).mockImplementation(() =>
```

Comment:
You can use the [`jest.mocked`](https://jestjs.io/docs/27.x/jest-object#jestmockedtitem-t-deep--false) utility helper.

```suggestion
      jest.mocked(pullAllTranslations).mockImplementation(() =>
```

Reply:
Oh yeah that's in jest now. Thanks.

Reply:
Gonna do a separate jest bump/cleanup PR as a followup. Still on jest 26.